### ISSUE TITLE
some questions about the training results

### ISSUE THREAD

**Issue by douyumo** [xmed-lab/UDCHealth#1]
{'jaccard_samples': 0.5072049804740842, 'f1_samples': 0.6550944694322899, 
'pr_auc_samples': 0.7689067343676638, 'roc_auc_samples': 0.9439015531365386, 
'precision_samples': 0.6977218020546678, 'recall_samples': 0.6645627939524793, 
'rec_grouped':
{'0%-20%': [0.470803539819712, 0.6253585612575577, 0.7423606276563623, 0.9308135164052006], 
'20%-40%': [0.4671636208688827, 0.6215321694234999, 0.742772228495414, 0.9328291960120084], 
'40%-60%': [0.48117584485914205, 0.6359873264191339, 0.756511658402762, 0.9366481533927805], 
'60%-80%': [0.4893471759861407, 0.6415622720657181, 0.7613395728302219, 0.9387618285379433],
'80%-100%': [0.5376679860502154, 0.6786820993307683, 0.7847411713736562, 0.9527632247721479]}, 
 'loss': 0.21110165600551667}
===============Aug Inference Done!===============
All Done!

Hello,Here are the results I obtained using your code. For the PCF pretraining model, I used the Transformer, as it seems to be the only available option in your implementation. However, the performance only meets expectations in the last patient group (80–100%). I would like to ask whether the metric reported in the paper is based on this final group.
